test/bun.js/jest.test.js

```diff
@@ -0,0 +1,43 @@
+import { beforeAll, it, describe, expect, afterAll } from 'bun:test';
+
+var before = false;
+var after = false;
+
+beforeAll(async () => {
+  if (before) throw new AssertionError("before should be false");
```

Comment:
Where is `AssertionError` defined?

Reply:
I might have misread the resulting `ReferenceError` as a sign this was working :sweat_smile: